modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/freelist/CacheFreeList.java

```diff
@@ -75,9 +76,4 @@ public CacheFreeList(
             "Constructed a link with invalid partition ID [partId=" + row.key().partition() +
                 ", link=" + U.hexLong(row.link()) + ']';
     }
-
-    /** {@inheritDoc} */
-    @Override public String toString() {
```

Comment:
Interesting, why did you remove this method?

Reply:
it exactly duplicates the method from its superclass

Reply:
Ok